server/routes/github/github_user_repos.py

```
@@ -30,148 +83,330 @@ def create_cors_response(data=None, status=200):
-                f"https://api.github.com/repos/{quote(repo_full_name, safe='/')}/branches",
+                f"https://api.github.com/repos/{repo_full_name}/branches",
                 headers=headers,
-                params={"per_page": per_page, "page": page},
-                timeout=10,
+                params={"per_page": _PER_PAGE, "page": page},
+                timeout=_GITHUB_TIMEOUT,
             )
-            
+
             if response.status_code != 200:
                 logger.error(f"GitHub API error: {response.status_code}")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)